src/hotspot/share/opto/macro.cpp

```diff
@@ -2457,6 +2457,13 @@ void PhaseMacroExpand::eliminate_macro_nodes() {
 //------------------------------expand_macro_nodes----------------------
 //  Returns true if a failure occurred.
 bool PhaseMacroExpand::expand_macro_nodes() {
+  // Perform refining of strip mined loop node before we start to expand.
+  for (int i = C->macro_count(); i > 0; i--) {
+    Node* n = C->macro_node(i-1);
+    if (n->Opcode() == Op_OuterStripMinedLoop) {
```

Comment:
You could also use `is_OuterStripMinedLoop()`:
```suggestion
    if (n->is_OuterStripMinedLoop()) {
```